src/checkin/CheckInQueue.jsx

```diff
@@ -124,6 +126,7 @@ class CheckInQueue extends React.Component {
 
 const mapStateToProps = (state) => {
   return {
+    error: selectors.isPatientStoreInErrorState(state),
```

Comment:
I guess all that Redux madness does afford one some very pleasant application-layer code